.travis.yml

```diff
@@ -0,0 +1,46 @@
+sudo: false
```

Comment:
Should we distribute this with marketplace-gulp as well since it's a file that will often update? I feel we should make the consuming projects as lean as possible.

(I'm still slightly thinking about distributing the Karma files since there's a chance we update those at some point as well.)